### ISSUE TITLE
Build ruff fails on FreeBSD

### ISSUE THREAD

**Issue by donnex** [astral-sh/ruff#13660]
Right now it's not possible to compile `ruff` on FreeBSD.

The build fails on `tikv-jemalloc-sys`.

This is problematic since `pip` tries to download and compile `ruff` if it's listed as a project dependency and this fails on FreeBSD.

This is probably the same issue that was recently fixed in `uv` at https://github.com/astral-sh/uv/pull/7780

[ruff-freebsd-build.log](https://github.com/user-attachments/files/17278006/ruff-freebsd-build.log)

**Comment by MichaReiser** [comment]
It seems that we intentionally enabled jemalloc on freebsd in the past. See https://github.com/astral-sh/ruff/pull/2965

**Comment by donnex** [comment]
You are absolutely correct. The fault was all mine because of `gmake` missing from my systems. Thanks :pray:
